Android/notification-hubs-sdk/src/main/AndroidManifest.xml

```diff
@@ -4,9 +4,9 @@
     android:versionCode="1"
     android:versionName="1.0" >
 
-    <uses-sdk
+<!--    <uses-sdk
```

Comment:
Why this change?

Reply:
Not able to compile with latest Android studio